Source/WebCore/Modules/modern-media-controls/controls/media-controls.js

```diff
@@ -187,6 +187,11 @@ class MediaControls extends LayoutNode
         return this.children.some(child => child.element.contains(tappedElement));
     }
 
+    reenable()
+    {
+        window.addEventListener("dragstart", this, true);
```

Comment:
This reenable() method isn't symetric to the disable() method. It's missing the reinstall of the "focusin" event listener, which is installed at creation time and uninstalled by disable() [here](https://github.com/WebKit/WebKit/pull/34350/commits/67bab4d4eee5640301ffc3c3a5132c7b50d3fbcc#diff-c73d8fa8d6e6c156a9d1b1871e8db6ca63a2686912354757011556091aff568fL75).

Reply:
You're right, I will submit something soon. I also moved the reenable function to a place closer to the disable one.